apps/api/src/domain/models/allowedContexts/isContextAllowedForGivenResourceType.ts

```diff
@@ -0,0 +1,36 @@
+import cloneToPlainObject from 'apps/api/src/lib/utilities/cloneToPlainObject';
+import { ResourceType, resourceTypes } from '../../types/resourceTypes';
+import { EdgeConnectionContextType } from '../context/types/EdgeConnectionContextType';
+
+const resourceTypeToAllowedContextTypes = {
```

Comment:
Add a test that ensures:
- there is one entry for each `ResourceType`
- the general context is registered as allowed for each `ResourceType`
- no entry contains duplicates 
- every context type is registered for at least one resourceType

Reply:
done